datafusion/physical-expr/src/expressions/case/literal_lookup_table/mod.rs

```diff
@@ -0,0 +1,440 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+mod boolean_lookup_table;
```

Comment:
Similarly to case I think it might be a good idea to move the bulk of this code out of mod.rs

Reply:
Ok for me.

Just as FYI, there was some discussion on this topic at https://discord.com/channels/885562378132000778/1435242236068495504 and https://github.com/apache/datafusion/issues/11375
No clear conclusion just yet.

I use 'jump to file by filename' all the time myself. A bit annoying when so many files are named `mod.rs`.